src/app/meme-card/meme-card.component.ts

```diff
@@ -23,7 +23,7 @@ export class MemeCardComponent implements OnInit {
       this.imageLink = meme.link;
       this.username = meme.creator["username"];
       console.log(this.imageLink);
-      this.totalVote = meme.totalVote || 0;
+      this.totalVote = meme.totalVote;
```

Comment:
This is the same?

Reply:
oh strange... I changed it on mine, but I thought I pushed it... well either way, we don't need the || 0 because total vote is default set to 0 now.